arrow-core/src/main/kotlin/arrow/core/extensions/set/foldable/SetKFoldable.kt

```diff
@@ -89,8 +89,9 @@ fun <A> Set<A>.fold(arg1: Monoid<A>): A = arrow.core.extensions.set.foldable.Set
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-    "reduceLeftToOption(arg1, arg2)",
-    "arrow.core.reduceLeftToOption"
+    "Option.fromNullable(reduceNullable(arg1, arg2))",
+    "arrow.core.reduceNullable",
```

Comment:
I just double checked this, and this is incorrect in `list` & `map`.
It's `reduceOrNull`, which is named after the Kotlin Std function.

Reply:
I'll change it for list and map as well 👍

Reply:
Thanks @danimontoya!